lessons/linear-regression/langs/en-us/pages/lr-plot-explore.adoc

```diff
@@ -0,0 +1,47 @@
+= lr-plot Explore
+
+**Part 1: You should already have plotted `lr-plot(animals-table, "name", "age", "weeks")` in the @starter-file{animals}... **
```

Comment:
It looks like you're using bold as formatting for a header. To be consistent with other worksheets, I would use `==` for a second-level heading

Reply:
@schanzer I tried your suggestion of using second-level headings, but I think reverting to bold would be better. Agreed?
[lr-plot-explore.pdf](https://github.com/bootstrapworld/curriculum/files/10045308/lr-plot-explore.pdf)
[lr-plot-intro.pdf](https://github.com/bootstrapworld/curriculum/files/10045309/lr-plot-intro.pdf)

Reply:
I think there's a use for both, but feel free to use bold for now